runtime/domain.c

```diff
@@ -18,10 +18,23 @@
 
 #define CAML_INTERNALS
 
+#define _GNU_SOURCE  /* For sched.h CPU_ZERO(3) and CPU_COUNT(3) */
```

Comment:
Is this only needed when `HAS_GNU_GETAFFINITY_NP` is defined?

Reply:
> Is this only needed when `HAS_GNU_GETAFFINITY_NP` is defined?

Yes and no, the problem is that in order to get `HAS_GNU_GETAFFINITY_NP`, I need to include `"caml/config.h"`, but I have to define `_GNU_SOURCE` before including `caml/config.h` since it pulls other headers that depend that break `_GNU_SOURCE` since it must be included before any things like `stdint.h` if you check `runtime/{domain.c,floats.c,unix.c}` they do the same.

Reply:
Got it, thanks!